src/api/bookApi.ts

```diff
@@ -33,15 +41,18 @@ export const fetchSearchBooks = async ({
         authorKey: author_key?.[0] || '',
         cover: docs[0].cover_i
           ? `https://covers.openlibrary.org/b/id/${docs[0].cover_i}-M.jpg`
-          : '',
+          : placeholder_book,
         isCurrentlyReading: false,
       }),
     );
 
-    return allBooks;
+    return { books: allBooks };
```

Comment:
IF you do
```suggestion
    return { books: allBooks, message: '' };
```

you can make the message required